readme/building_and_running.md

```diff
@@ -9,34 +9,38 @@ First, build the project by running:
-* [keyworker-api](https://github.com/ministryofjustice/keyworker-api) - an API providing backend functionality for keyworkers
-* [whereabouts-api](https://github.com/ministryofjustice/whereabouts-api) - an API providing backend functionality for the location of prisoners
-* [offender-case-notes](https://github.com/ministryofjustice/offender-case-notes) - an API managing storage of offender case notes
-* [hmpps-incentives-api](https://github.com/ministryofjustice/hmpps-incentives-api) - an API managing offender incentives data
-* [curious-api](https://github.com/ministryofjustice/curious-API) - Curious is a 3rd party supplied system for Education Delivery Contract Management
+* [prison-api](https://github.com/ministryofjustice/prison-api) - the main API for retrieving data from NOMIS
+* [hmpps-prisoner-search](https://github.com/ministryofjustice/hmpps-prisoner-search) - an OpenSearch backed API for
+  quickly retrieving commonly requested prisoner details
+* [offender-management-allocation-manager](https://github.com/ministryofjustice/offender-management-allocation-manager) -
+  allocation service for associating Prison Offender Managers with Offenders
+* [keyworker-api](https://github.com/ministryofjustice/keyworker-api) - an API providing backend functionality for
+  keyworkers
+* [whereabouts-api](https://github.com/ministryofjustice/whereabouts-api) - an API providing backend functionality for
+  the location of prisoners
+* [offender-case-notes](https://github.com/ministryofjustice/offender-case-notes) - an API managing storage of offender
+  case notes
+* [hmpps-incentives-api](https://github.com/ministryofjustice/hmpps-incentives-api) - an API managing offender
+  incentives data
+* [curious-api](https://github.com/ministryofjustice/curious-API) - Curious is a 3rd party supplied system for Education
+  Delivery Contract Management
 * [pathfinder-api](https://github.com/ministryofjustice/pathfinder-api) - an API for retrieving Pathfinder data
-* [manage-soc-cases-api](https://github.com/ministryofjustice/manage-soc-cases-api) - an API for retrieving SOC case data
+* [manage-soc-cases-api](https://github.com/ministryofjustice/manage-soc-cases-api) - an API for retrieving SOC case
+  data
 
 ### Developing against the development environment
-Development of this application has mainly relied on configuring `hmpps-prisoner-profile` to point at the development 
+
+Development of this application has mainly relied on configuring `hmpps-prisoner-profile` to point at the development
 environment instances of the above dependencies (redis being the exception, a local instance of this was used).
 
 Here's the process.
 
-1/ Run redis locally using Docker:
-```
-docker-compose pull && docker-compose up
-```
-
-2/ Create a .env file with environment variables pointing to the development environment
+1/ Create a .env file with environment variables pointing to the development environment
```

Comment:
...here, removing instruction to run redis locally (since no longer need to do this)